packages/ui/src/components/w3m-modal-toast/styles.ts

```diff
@@ -9,14 +9,16 @@ export default css`
     justify-content: center;
     align-items: center;
     padding: 10px 15px;
-    border-radius: 36px;
     position: absolute;
-    bottom: 60px;
+    top: 12px;
```

Comment:
We should really look into more sturdy styling, absolute position always manages to break at some point

Reply:
I agree, but I'm not sure if there is way around this here, as this toast thingy needs to "cover over" main content